cpp/src/link_analysis/pagerank.cu

```diff
@@ -208,7 +208,7 @@ gdf_error gdf_pagerank_impl (gdf_graph *graph,
     GDF_REQUIRE(personalization_subset->null_count == 0 , GDF_VALIDITY_UNSUPPORTED );
     GDF_REQUIRE(personalization_values->null_count == 0 , GDF_VALIDITY_UNSUPPORTED );
   }
-  GDF_REQUIRE( graph->edgeList != nullptr, GDF_VALIDITY_UNSUPPORTED );
+  /*GDF_REQUIRE( graph->edgeList != nullptr, GDF_VALIDITY_UNSUPPORTED );

```

Comment:
-> Remove edgelist checks since user input could be in the form of adjacency list(edgelist might be nullptr in that case)
-> Remove pagerank column checks since the column is initialized in the pagerank_wrapper.pyx with pre-set specifications(size, dtype, data initialized to zeros, etc) which is redundantly verified in cpp.
Note: Commented lines to be removed after initial review.

Reply:
I agree regarding dropping edge list checks but I think we should keep PageRank ones in case someone calls it from the c++ API. It will also help debugging if we introduce an issue at the python level.